failsafe/circuit_breaker.py

```diff
@@ -40,14 +40,14 @@ def allows_execution(self):
 
     def record_success(self):
         """
-        Stores an execution success in the state of the CircuitBreaker
+        Function called when the execution succeeded
```

Comment:
If we add this explanation here, maybe we can delete the docstrings for the ClosedState and OpenState `record_success` and `record_failure`.

Instead of that, in my opinion, we could have here something like this:

`Records an execution success - should be called when operation protected by circuit breaker succeeded`

And in the subclass, the part of:

In ClosedState:

`Resets the counter of consecutive failures.`

And in OpenState:

`Does nothing.`